### ISSUE TITLE
 [Request to become a PrestaShop maintainer] PrestaEdit

### ISSUE THREAD

**Issue by PrestaEdit** [PrestaShop/open-source#106]
And here we are.

## A little bit of history

- 2010, August : My start with PrestaShop was made.
- 2012, May : I made a soft called "PrestaEdit" to manage a PrestaShop project as student project.
- 2012, August : the paper "PrestaShop, aspects techniques" is published in the _PhpSolution_ magazine.
- 2012, September : the paper "PrestaShop 1.5 : tour d'horizons et aspects techniques" is published in the _PhpSolution_ magazine.
- 2012, November : PrestaShop is now on GitHub. And my first pull requests are done the same time. (As @kpodemski, my first one, the https://github.com/PrestaShop/PrestaShop/pull/2/, was wrong :D)
- 2012, December : first (and only one before moving to GitHub) external contributor to the french documentation of the 1.5 version. About this : I could manage to make some captures, still having the credentials.
- 2013, April : my first big pull request at this time, not merged but the one that make me create modules after it. https://github.com/PrestaShop/PrestaShop/pull/90
- 2013, May : The "PrestaShop s'envole sur GitHub" blog post was made on PrestaShop blog with an interver of me as contributor.
- 2015, June : winner of the PrestaShop Awards "Contributor Favorite, Most line of code modified".
- 2015, July : First PrestaShop Ambassador.
- 2015, August : Joining the Presta-Module team as module developper.
- 2019, January : Quit the Ambassador program due to internal change and I assume that I wasn't able to stay at this role at this time.
- 2022, June : The contributor intervew was publish on the Build blog.

## And now ?

Okay, it's a lot of history and we need to be focus on two things : now and future.

Now, I'm still a PrestaShop module developper inside the Presta Module, Business Tech and Seo Presta team.
Also, since june of 2022, I'm not in charge of the management for the Presta Module catalogue.

In the same time, I'm writing a book (in French) to develop with PrestaShop 8.

Also, if you check the TopContributors board, you will see that I have 499 contributions. https://contributors.prestashop.com/#PrestaEdit (You can also check https://contributors.prestashop.com/#prestamodule with its 124 contributions, a big part are mine ;)).
But, to be honest, it's not what I'm checking when regarding my contributions.

Mines are everywhere, like Twitter or Slack.
Also, it's not only with commits or pull requests.

<img width="765" alt="Capture d’écran 2022-06-24 à 12 20 53" src="https://user-images.githubusercontent.com/2631425/175515879-449a4156-e61e-4f7c-8769-211382038202.png">

## Why, and why now ?

It's the main ask that we could have.

Why ? Because PrestaShop is right now part of my life, having 35 years old. It's like a third part of it since my beginning and I always give my self as possible to help the project or, especially, the community.

Why now ? Community. It could be strange to read, but even if I was having a 1,5 year period where my implication was not enough (for me) on this project, the community is a big part and it's the community that make me want to be there for it.

My mentor, as PrestaShop Maintainer, is @Progi1984. My application is here, right now, because of him. (Of course, because of me, as I want it, but he help me a lot to start it.)

## Asks

Regarding your process (sorry, @matthieu-rolland, you will not be able to copy/past it there 😌), I'm here for your asks.

**Comment by Progi1984** [comment]
It's (of course) an approval on my side.

**Comment by jf-viguier** [comment]
I can't but I approve too

**Comment by kpodemski** [comment]
It's great to see your application. You are one of the _dinosaurs_ of this ecosystem, and so am I :D because I remember you from yeeeeears ago. Big person of our community.

The one thing that is important to know before becoming a maintainer is that **you have to be a team player** and **respect others' decisions** that you might disagree with. We won't agree on everything. You won't have your ideas or pull requests prioritized. I'm writing this because I remember some heated discussions between you and other maintainers (me included) where your reaction was quite harsh.

If I reject your changes and you do not agree with them, will we have a productive discussion or we'll fight with the use of the GitHub comments? ;-) I had to answer a similar question because I was in pretty much your position. As a module developer, I think I know better, where sometimes the situation isn't as black and white as we think.

**Comment by PrestaEdit** [comment]
> Will we have a productive discussion or we'll fight with the use of the GitHub comments? ;-)

Great ask, and I assume why it's there :D

You know better than no one that I'm fighting for my ideas and be able to understand a good reason to not agree with them.
The only things I need, and so what I will do more in a future, it's to clarify this reason (as you know, some times the product vision is not clear enough and it could be difficult sometimes to understand a negative review to something while you are not in the same mind).

Of course, I'm more able to work on the legacy part (even If the new architecture is something that I follow, work on) and be able to clearly see a potential issue or bug with themes (as https://github.com/PrestaShop/PrestaShop/pull/26225) or modules.

But I don't want to be prior : it was I fighting since a long time in this ecosystem.
Just want to be there to give my ideas, understand anothers one and agree with them even if I'm not in the same mind at the beginning.

Also, be there for future contributors, to be there as @Progi1984 could be : an helper (not an HelperForm, of course :D)

**Comment by matks** [comment]
Dear PrestaEdit

I have no doubts regarding your prestashop skills. I have some questions though:

1. I see many of contributions have a branch named `patch-(...)` which are often indicators that PR was created using GitHub UI and not git tool. Are you confortable with the git tool? Being able to rebase, cherry-pick, handle git conflicts?

2. Just like Krystian said I think I remembered some discussions where you and I disagreed. Are you ready to follow the group's goal even if you don't share it? Because if you become a maintainer you become a representative of the project. When you speak and write on GitHub you become one of the many voices of the project. We need to know if we can trust you and give you this responsability, aware that we cannot verify everything you write. We need to be able to say "I trust PrestaEdit, he can speak in my name, I know he won't purposefully say something I would be strongly against"

3. About this item:

> Why now ? Community. It could be strange to read, but even if I was having a 1,5 year period where my implication was not enough (for me) on this project, the community is a big part and it's the community that make me want to be there for it.

For many years I have wondered why some people, me included, would disagree with others _although we all think we work for the community_. I think you believe you work for the community. I think I do too.

Recently I started to think: maybe it's because "the community" is actually not the same thing for you, me, and others. I now think there are many communities in PrestaShop ecosystem. For example:
- GitHub contributors community
- Module developers community
- Web Agency developers community
- Sysadmins community
- Small merchants community
- Big merchants community

And all of these people have different goals and sometimes these goals go against each other.

For example, the module developers community probably wish that it is easy to have a module compatible with PS 1.7 and 8.0 . So they would be against Breaking Changes introduced in PS 8.0.0 (including removing deprecated code) because it makes their work harder. They would also need that PrestaShop has a php compatibility as wide as possible.

The agency developers community probably don't care about php compatibility as wide as possible: they just need one, the one they run in production. And even though BC breaks make upgrades harder to do, they might welcome BC breaks more willingly because they only run one version of PrestaShop.

But there is more! Even in these communities they are subgroups. For example the GitHub contributors community contains at least 4 subgroups I know:
- people that consider current PrestaShop stack good enough and prefer we iterate on it (don't modify the frameworks or "things that already work", fix bugs, no BC breaks) - for example I think @hlavtox is in this group (@hlavtox please forigve me if I am wrong)
- people that consider PrestaShop stack is old and want it to be more modern, it could be because they see it becomes hard to hire developers to work on PrestaShop (because developers, especially juniors, want shiny new things) or to sell PrestaShop or because they simply enjoy working on state of the art technologies like React, Symfony, GraphQL - for example I think @kpodemski is in this group (@kpodemski  please forgive me if I am wrong) so they challenge PrestaShop frameworks and years-long practices and wish for big moves soon
- people that run PrestaShop on a server as stand-alone
- people that deploy PrestaShop on multiple instances

The ["Bump minimum PHP version for PrestaShop 8.x ?"](https://github.com/PrestaShop/PrestaShop/issues/27829) is a good example of multiple people of different communities wanting different things. But they all are members of the PrestaShop community, however they belong to different subgroups.

Back to your application: I'm sure you will "defend" the interest of your community but are you ready to consider other communities needs and follow the group's will when it does not go your way?

Basically we can reduce my items 2 and 3 to: are you willing to be a team player and play with the team even though sometimes the team will go _against_ what you think is best? And sometimes the opposite will happen: for example I will say "we need to do X" and you will say "no we need to do Y" and the maintainer team will choose Y instead so we'll do Y

**Comment by PrestaEdit** [comment]
Hi @matks, and thanks for your message.

There is somes answers to your asks : 

1. A lot of my branch are manage as this because I manage to make the patch only on one file while needed. If I need to impact more files and/or need to make more commits in order to facilitate the read of the Pull Request, I will use the GitHub Desktop (and git cli) tools.

If I'm honest, I need to rework the way of working with git cli and somes commands that I don't use everyday. Even if I'm not a maintainer, is something on what I will have a look sooner.

2. As you know and as I said to Krystian, I will every defend my position but if the main vision is clear, I will respect it.
My main issue while discussing with someone on the PrestaShop vision is that... the vision is not clear, not discussed. Only the fact that It will be not make as this or as that will be. I will appreciate something like the PHP Votes : of course, not everybody could vote but the discuss are shareable and so the vision is clear.

3. I will not be longer: community, it's everyone. It's who make PrestaShop, even the haters. It's also the PrestaShop Corp.
It's you, it's me. It's agencies, it's modules developpers (I open this right now : I'm a module developper and... the man that make a lot of deprecated removes since the beginning, :D), it's merchants. Everyone that use or talk about PrestaShop is part of this community.

Regarding your ask on this part : As you could see, I also make somes Pull Requests on issues that are part relative to something I don't use even one day a month. Because somes are using them and need to have a change on them.

All is about communication : if you are a team player, I'm. If I'm able to say that the team (even if it's only 60%, for example) go in the same way than you, I can accept it even If I disagree on my own idea.

**Comment by matks** [comment]
😄 I got all my answers thanks

I added some extra information so that you have some context about what's going to happen if you join us 😄

**Comment by PrestaEdit** [comment]
Good,

Is someone else have somes aks, don't hesitate. I'm here to answer them.

**Comment by matks** [comment]
@PrestaEdit we let some time for everybody to make up their mind then we launch the vote ;)

**Comment by matks** [comment]
Hello @PrestaEdit I was very late on pull request review recently so today I did a big number of reviews. And I found multiple topics where you started something but you could not finish 😅 

https://github.com/PrestaShop/PrestaShop/issues/28813
https://github.com/PrestaShop/PrestaShop/pull/28199

And I think to myself ... 😅 if you don't have time to make your PRs / issues go forward, will you be able to find time for maintainer role?

**Comment by PrestaEdit** [comment]
Hi,

Don't your answer youself to the ask ? As you say, you was on late and be able to take time now.

Sometimes, while you do a lot of of thing and have multiple feedback not on the good time, you could loose a notification.
For that point, I dislike GitHub and I need to find a way to improve it's management on myself.

(Do I have to, really, point some others issues/pull requests without no answer since a time about maintainers or, simply, I will assume that I have a job, two childs, and that I can understand it's the case for everyone and that sometimes you could forgot to take back something you have started and/or it's not the right moment ? Mmmmmh. Yes, I will not doing it, of course.)

**Comment by Hlavtox** [comment]
@matks I would not generalize like this bro. For example - I am stuck on github non-stop during winter time, but I don't have much time during the summer. Sometimes you just don't have a mood to finish a PR that you started, or skills, and let it open so that you don't forget.

I think we can be glad of every hand of help the project can have, if we want to catch the train.

**Comment by PrestaEdit** [comment]
Hi everyone,

Do you have some news on this, maybe ?

As I know, it's July and free time is there for everyone especially with holidays, so, I fully understand the needed time in each case.
Just for me to know, if you have some asks or something else where I can assist you.

**Comment by eternoendless** [comment]
Hi @PrestaEdit and thank you for your submission. So sorry for the lateness, with holidays, emergencies to attend and all. 

Before anything, I'm sure you can bring a lot to the project, and I believe most maintainers would probably say yes if we started voting. 

My question is, **what do you expect out of being a maintainer?** I don't mean that in the sense of what "benefit" do you expect to get out of it – that's none of my business really. I mean, what do you expect you will be able to do within the project that you can't do now?

In particular, **why do you want to be a maintainer instead of a committer?**

I have given the subject some thought recently. We face contradicting needs. On one hand, we want to get as many people as possible involved in the project, as quickly as possible. This means that we want to make the process of contributing actively as seamless as we can. On the other hand, we also want to make sure that the quality level and the processes we so painstakingly put in place over the years don't go out the window. So we want to be careful about who we give the "keys" to the project to. 

We feel naturally cautious when giving out the maintainer role, because it carries out a big responsibility. Maintainers have a vote on architecture decisions, and can vote others maintainers in. We need to trust the person we give that role. At our company, we test candidates before we hire them, and we have them go through 3-6 months of full-time close collaboration before they can apply to become a maintainer – even though they are bound by employment contract! How could we build that kind of trust from people from the community that we rarely work with? It would take ages.

In addition, we have experienced maintainers who either weren't ready for the role, or that expected more from it, and got frustrated. Being a maintainer means being a leader and acting like one, but also accepting that you also have to take care of the nasty stuff, and you don't get to decide on everything – and especially not on your own.

This is why we created the committer role. It's a role that allows contributors to start getting more deeply involved in the project, but with lower stakes: committers don't get to vote on who joins or leaves the project, nor on architecture decisions. But they help get Pull Requests approved, and move the project forward. Since committers can't approve PRs on their own, it's less risky to onboard people that we don't fully trust yet. And since the rules are clear, there's less risk of frustration.

And this is the point I wanted to get to. The project's decision-making process is still not clear enough. The company's role in the decision-making process isn't clear enough. Hell, my own role as the project's "dad" is not clear enough. It's difficult to engage the community this way. It's frustrating.

The good news is that I have been given the authority to work on these subjects personally. In the coming months, we will start clarifying the different roles and how they work together, how decisions are made, and how to integrate the community better and better in the project's day-to-day life. By working closer and closer together with people from the community, we will build trust. And having built trust, maintainership (or any other position of power/responsibility) will simply become the natural evolution of an engaged project member. Even if we might need to redefine what a maintainer is.

I think that today, the best way to become a maintainer and be included in the project's life **is by becoming a committer first.** What do you think?

**Comment by PrestaEdit** [comment]
Hi Pablo,

Glad to read you, expecting that it was a free time for you in theses particular days.

I will answer to the main ask, even if as you say it's not really something on wich we need to talk about.
As before and as never, benefits its simple and clear : no one.
As a module developper, I never try to change PrestaShop to fit the business. I always change my way to achieve them fitting PrestaShop.
But, yeah, if I can make one or another pull requests to allow some new features or fix some of them, it's a good point.

Let me now answer to the second - and most important - ask inside your long message : being maintainer instead of committer.
As far I know, committer will review pull requests and mark them as valid if needed.
Do I need to be committer to review a pull request ? No, not really.
Does the project need me as a committer to evolve ? No, not really.

As long I was involve inside the PrestaShop project (not the company, the project), I'd never need to have an attributed role to be part of : documentation, tests, contributions, ... 

So, why ? Cause it's pretty clear that I doesn't that to fell free to continue to be there and invest my time for that, as long as I can.

You surely more than me, especially now, that the project it's moving, changing.
The company role is less than before (or it's say that its).

And so the project too.

I've known many of core developpers, or managers of PrestaShop.
Finally, the only one to ask me to be part of decision sometimes was Bruno Leveque himself, when he was working on the Dashboard feature for the 1.6 version. This was on june 2013.

Since that, decisions are made by... - there is a lot of people to mention there, but I'm pretty sure somes will ever don't know who they are and it's not needed to.

Main reason is especially the point that define maintainer : be part of the decision inside the project.
And be part of them outside the company. Right now, as I can read you, it's pretty clear that even if PrestaShop say all time that the govenance need to be there, it's more : yes, but with somes of us as the top of it.
And it's what I want to does not see there.

As a PrestaShop contributors for many years, as a PrestaShop lover, I can't imagine to let the project moving forward and see it go into a way that I can regret without being part of the changes.

Hoping that everyone will read carefully my answers since the beginning and what's motivate me in this request.
Hoping that people will not vote yes only because of me, because of my knowdledge or because they feel that the project doesn't move fast and need only a hand to validate work from another.

I will ask to everybody for which the vote count to transform it to a no if it's the reasons for them to voting or simply to avoid voting.

Being maintainer it's not only a role. It's a way of thinking.

**Comment by kpodemski** [comment]
Hi @PrestaEdit 

Thanks for your detailed answer. I have a few thoughts after reading this.

> Do I need to be committer to review a pull request ? No, not really.
> Does the project need me as a committer to evolve ? No, not really.

First, I am a bit surprised you see no value in a committer role. I think our current committers show how they can help move the project forward. They are doing a fantastic job assisting maintainers in handling Issues, Pull Requests, etc. It is a big help and something that is not entirely possible without proper rights. I really appreciate this role, and I think we need more people interested in joining us.

I would like to precise one thing. I would like to know if you understand the maintainer role correctly. I'm referring to this part:

> Main reason is especially the point that define maintainer : be part of the decision inside the project.
And be part of them outside the company. 
> As a PrestaShop contributors for many years, as a PrestaShop lover, I can't imagine to let the project moving forward and see it go into a way that I can regret without being part of the changes.

Are you aware that our role is to help shape **technical side of things**? We don't have complete control over the entire project, Product-wise or business-wise. Maintainers don't decide on everything that's going on in the project. I wouldn't want you to be frustrated because of this fact.

**Comment by PrestaEdit** [comment]
Hi,

About committer role, it's not was I said. As you need to know, I found valuable every role : maintainers, committers, contributors, QA, somes that make issues, somes that even use the soft. All of theses roles are valuables.

I need to, now, asks you somes and does not only give answers.

So, who decide ? When, why and where they come from ?
ADR are part of theses decisions, like having a roadmap (at least a technical roadmap as migrate Symfony, having Back Office using it entirely and so on...).

Who will decide of this ? Are you able to discuss on this subject with maintainers, at least ?
Or do I need to say, now, that it's pretty clear that the governance of the project is not so open as wanted/said since months, now ? Is there somes votes on theses subjects or do you receive somes needs coming from PrestaShopCorp and need to be compliant with ?

**Comment by kpodemski** [comment]
Ok, sorry. Maybe I misinterpreted it. I believe being a committer first is an excellent way to get to know the project organization, work closer with maintainers, etc. It is not only about knowing PrestaShop core.

Governance. It is still something that people in the organization are working on. You see how the project is evolving, but it is not a library maintained by three people. It is a big project.

You can see many records of decision-making, on GitHub. You can see comments from the product team, ADR, discussions with QA, and the maintainers team.

I think @eternoendless answered this already:

> And this is the point I wanted to get to. The project's decision-making process is still not clear enough. The company's role in the decision-making process isn't clear enough. Hell, my own role as the project's "dad" is not clear enough. It's difficult to engage the community this way. It's frustrating.
> The good news is that I have been given the authority to work on these subjects personally. In the coming months, we will start clarifying the different roles and how they work together, how decisions are made, and how to integrate the community better and better in the project's day-to-day life. 

This is why I would be more than happy to see you as a committer. Then, after clarifying the responsibilities of maintainers, you could decide if it is something you are interested in.

**Comment by matks** [comment]
One of the key of open source is to be transparent, so I'll be transparent and see what's going on in my mind. Please be sure I want to hurt nobody 🙏 however I must say that I am now worried, because of the last items of the discussion. Also I'm going to present my own vision of this project and the maintainer group but it's only mine, I have no idea if the others share it 😅 .

Discussing online is very hard because we only see the words, we don't see the feelings, the tone, the faces so maybe I am wrong but I feel like you're focusing very much on "power", "decision", "control" (this is why I add so many emojis 😄 it helps to show the mood behind the words).

If indeed this is the main focus, I am afraid this is not a good reason to become maintainer 😞 in my opinion.

It is a personal opinion but for me decisions and controls are simply tools we use when there is a problem where we need to act as a group.

I am maybe naive but here is how I picture this maintainer group: a group of friends (= maintainers) decided to build a house together (= the project). Building a house takes months of work. The main thing the friends are going to do is ... the building part. Sawing, pressing, shoving, bearing. Sweating, pushing 💪 . This is 90% of the work.

Sometimes, the group of friends have to make a group decision. For example do they put the bathroom on the left or the kitchen? When there is a clear group agreement (= everyone has the same opinion), no need for vote or decision-making, they just go with it. However sometimes part of the group has an opinion, the other group thinks differently.

This is where the decision-making is useful, it helps to find what to do in such situation. It helps solving the problem and it says "We are going to follow {...} process (can be a vote, can be an election, can be random pick?) and it will tell us what we do" 🔧 

As you see, the decision-making for me is just a tool that avoids the group to be stuck when there is disagreement. A way to avoid dead ends. It allows to continue moving forward.

If tomorrow another person comes and says: "Hey, I want to join the group I wish to see this house built too! I want to help" (= submits application to become maintainer) this is wonderful ❤️ . But the main reason behind, for me, should be "This house building project is important for me I want it to be done I want to be part of it".

But when I read these last items, I rather imagine another person that comes and says: "Hey, I want to join the group because I don't want it to be done in a way I regret so I want to join to have decision power when decisions are made". Woups 😅 it does not sound right. It sounds like bringing this person in the group might bring disagreement, division, maybe disharmony? Then... is it a good idea? Is it a good thing for the group? Is this going to help the house be built faster and better or is it going to reduce the group ability to work in harmony? 🤔

**Comment by kpodemski** [comment]
Hi @PrestaEdit 

Did you have time to consider what we've written above?

**Comment by PrestaEdit** [comment]
Hi,

Yes, @kpodemski.

As I was with my sons and sick last days, I didn't answer.
Althougt, I think is more an opinon or a change you want to make on the govenance project and not something on wich I need to answer or to debate there.

I read, I unsterdstant it. I let you choose on your own how to take care of the previous messages in your vote.

**Comment by jf-viguier** [comment]
Any news here ? Two months for having a vote ?

**Comment by eternoendless** [comment]
Sorry for the delay. As you might imagine, we are currently cautious about new applicants because people don't seem to grasp what being a maintainer is actually about, and that leads to frustration.

In this particular case, I'm worried about your motivations.

> Main reason is especially the point that define maintainer : be part of the decision inside the project.
> And be part of them outside the company. Right now, as I can read you, it's pretty clear that even if PrestaShop say all time that the govenance need to be there, it's more : yes, but with somes of us as the top of it.
> And it's what I want to does not see there.
> 
> As a PrestaShop contributors for many years, as a PrestaShop lover, I can't imagine to let the project moving forward and see it go into a way that I can regret without being part of the changes.

Please correct me if I'm wrong, but this is what I understood from the excerpt above: you think that PrestaShop SA wants to keep the project's governance for itself. You disagree with the company's vision, so you want to join the maintainer's team to try and wrestle the project in a different direction.

That doesn't sound like a constructive relationship. 

You don't need to join the maintainers to voice out your concerns, they will be heard regardless of your status within the project. Being a maintainer is less about directing the project, and more about **actually moving the project forward**. How? By reviewing PRs, by contributing important fixes and strategic features that no one else will do, by helping other contributors solve problems, by discussing with maintainers to find the best tech choice. Maintainers decide by voting on implementation (via code review) and general architecture decisions. ADRs are records of those decisions. 

Now, regarding PrestaShop SA and governance:

Currently, the project is defacto led by the company because the company does the overwhelming majority of the work. We want this to change by removing barriers to contribution. For that, we need committers, maintainers, and other roles like product managers and QA that will be clarified in the future. In time, having a more evenly distributed contribution load will naturally require stronger cooperation. This is where open governance kicks in.

Open governance is not about having people decide what other people will do; that won’t work. People will only work on something if it’s important for _them_. The whole point of opening up the governance is about establishing a cooperative environment that enables groups of individuals to work on what they need, while ensuring that the project doesn't become anarchic and keeps moving forward towards a common goal.

A person might have the best ideas, but to get them done they will need either the resources to do them on their own, or the influence to persuade others to build them for them. No artificial status or title (maintainer or otherwise) will grant that power automatically.

Going back to your application: I'm not sure you're ready for the maintainer role just yet, but I think you could be a great committer.

**Comment by PrestaEdit** [comment]
Hi @eternoendless ,

I can assume that your reads if not the good one, about my motivations.

I'm not against PrestaShop SA and the move they want to do, espacially when I'm not implicated in theses moves (and so I doesn't know them).

Also, as I can read since weeks/months is that I'm the only one (or approximatively) to be honest and eyes opened about this governance : PrestaShop SA will have yet something to say, something to do on this project. And, yes, I'm thinking it's quite normal but it's need just to be clear enough for everyone.

When I say that I want to be part of the project especially outside the company is about the fact that I will make PrestaShop SA like others users of PrestaShop. It's like changes inside our own companies, we does not wait something to be move on the project to make decisions. It's not for that reason that I will say to another company or to the project to follow theses.

The only part that I really care about is this one :
> Maintainers decide by voting on implementation (via code review) and general architecture decisions. ADRs are records of those decisions.

Going back to my application: I'm not sure PrestaShop is ready to extends the maintainer role just yet, but I think you could clarify this and just say the words.

To be honest, I don't understand how this is going and how it's work.

I let you make the things that you need to do, that the PrestaShop project need.
Not more, not less.

There is also no need to waste time or take it for nothing : the maintainers are now 9. There are at least 3 negatives votes, as far I know. The result is that the two-thirds positive votes is not achieved.

As I assume that the vote is not anonymous and if you don't want to know how others maintainers will vote, don't bother them.

**Comment by PrestaEdit** [comment]
Hi,

Any updates, there ?

**Comment by Progi1984** [comment]
Ping @PrestaShop/prestashop-maintainers

**Comment by matks** [comment]
> Hi,
> 
> Any updates, there ?

Hello @PrestaEdit you're right to ping 👍 . As the messages above, this application actually brought a lot of questions that needed to be explored before we could vote. This is the main reason why this application was submitted in June but only now the vote happens. Also summer 😅 

But a vote is needed and you're right to remind us: I think most maintainers now feel like they fully understand your application, motivation and reasoning and this allow them to vote appropriately as all the needed items are there.

The vote has started. As some of us are less active than others, we'll provide multiple days for maintainers to bring their vote but I think no more than 2 weeks should be needed to have an answer.

Have a nice day ✋ regards

**Comment by matks** [comment]
Dear @PrestaEdit the vote is complete.

The [nine Project Maintainers](https://www.prestashop-project.org/maintainers-guide/project-organization/) all provided the same answer to your application:

We do not think _now_ is a good time for you to become a maintainer.

However we think you could be a great [committer](https://www.prestashop-project.org/maintainers-guide/how-to-become-a-committer/) (you're already providing code reviews on the project Pull Requests).

We think that if you become committer this would be a great way to involve you in the project and make you ready to become a maintainer. You can see it as a step on the path to maintainer.

The nine maintainers said that they would be happy to welcome you as a committer so if you tell us that yes you wish to become committer, this could happen immediately.

**Comment by eternoendless** [comment]
@PrestaEdit I'm under the impression that we'll both be attending FoP day in November. If that's the case, I hope we can get a moment to discuss this subject IRL and find a way to get you more involved in the project.

**Comment by kpodemski** [comment]
@PrestaEdit 

I hope you'll reconsider joining the project as a committer because it is not only about being in the specific group of users on GitHub.com. There's more to it so that we can work together more closely.

**Comment by jf-viguier** [comment]
After 3 months, it's a sad day for the PrestaShop community.  IMO good wills should be better considered in a shorter and simpler decision process.

**Comment by eternoendless** [comment]
I'm sorry that the final decision took so long. But if it did is because saying no is hard, and it is especially hard because we appreciate your work, and we definitely want to have you as involved as possible in the project – even if now it's not the right time for you to become a maintainer.

I was looking forward to meeting you at FoP day, that's a shame. The door remains open for you to become a committer. Feel free to reach me on Slack if you change your mind.
